modules/bogo/includes/REST/BogoController.php

```diff
@@ -186,6 +186,9 @@ protected function get_total_items_count( $query_filters, $request ) {
     public function get_items( $request ) {
         // Get custom query filters and options from child classes
         $query_filters = $this->get_query_filters( $request );
+		if( ! current_user_can('manage_options') ) {
```

Comment:
Pls remove this section.

Reply:
- Use `spsg_bogo_query_args` filter for query
- Use `spsg_bogo_created_by` filter  for vendor despite the staff.